requirements_tests.txt

```diff
@@ -1 +1,2 @@
+tensorflow_probability==0.7.0
```

Comment:
Can you not pin 0.7.* instead? Pinning an exact minor version is very strict.

Reply:
Right, of course I can do that for now. But as I have said in one, I will try to set up a conda environment for the testing pipelines to avoid those compatibility problems with tensorflow, tensor flow probability ...

Reply:
Right, forgot that.